### ISSUE TITLE
Lifetime error

### ISSUE THREAD

**Issue by sander1** [plexinc-plugins/Services.bundle#872]
URLs that redirect will not work (videos are gone or moved to Hulu) and throw an error.
Example: http://www.mylifetime.com/shows/unreal/video/season-1/episode-2/relapse

**Comment by shopgirl284** [comment]
This should no longer be an issue now that we have changed them all to use the aetv API pages instead of the website links. That API is much more consistent in the URLs it provides only being those that are unlocked and properly formatted.

The example URL given is no longer available so we cannot test that one. 

I did find one a "catch-up" video that does not give a proper URL and fails. The API returns http://www.mylifetime.com/shows/unreal/video/season-2/episode/unreal-quickie-catch-up, but the actual URL is http://www.mylifetime.com/shows/unreal/season-2/episode-0.

This can be fixed by cchecking for special episodes that do not have an episode number and adding a "-0" to the URL. Like adding something to the NormalizeURL function that checks for "/episode" that is not followed by a "-" and replacing it with "/episode-0"
